packages/desktop-client/src/components/spreadsheet/CellValue.tsx

```diff
@@ -32,7 +35,8 @@ export function CellValue({
   privacyFilter,
   'data-testid': testId,
   ...props
-}: CellValueProps) {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+}: CellValueProps<any, any>) {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏anything we could do here in order to get rid of the `any`s?

Reply:
I believe this will be fixed in part 3. I wanted to have looser types in the interim so we could split up the changes.

Reply:
That reminds me though, I should go and rebase part 3 on this PR again so the change is clearer compared to this PR.